electronicparsers/openmx/parser.py

```diff
@@ -744,6 +816,19 @@ def parse(self, mainfile: str, archive: EntryArchive, logger):
             self.logger.warning('.md file does not contain enough MD steps')
 
         if mainfile_md_steps is not None:
+            if is_stdout_valid:
+                mainfile_uele_str = mainfile_md_steps[0].get('SCF')[0].get('Uele')
+                mainfile_uele_float = float(mainfile_uele_str)
+                mainfile_uele_precision = num_of_decimal_places(mainfile_uele_str)
+                desired_precision = min(mainfile_uele_precision, stdout_uele_precision)
+                offset = None
+                for index, stdout_uele in enumerate(stdout_uele_list):
+                    if is_close(float(stdout_uele), mainfile_uele_float, desired_precision):
```

Comment:
and here replace with abs(stdout_uele - mainfile_uele) < 1e-8

But I don't really follow this loop, aren't you comparing the 0-th value of Uele  from the main *.out file with all of the Ueles from the stdout? I think we should just compare the first (or last) ones, or alternatively compare all of them, but always i-th with i-th.